godmode.txt

```diff
@@ -0,0 +1,44 @@
+GodMode User Notes
```

Comment:
since we have started to comment on this file, I will change this to an md file after the necessary changes